website/docs/components/modal/partials/code/how-to-use.md

```diff
@@ -63,7 +69,10 @@ When the Modal dialog contains information that might be lost on close, use a co
       Why do you want to leave the beta?
     </M.Header>
     <M.Body>
-      <form name="leaving-beta-form">
+      <form
+        id="leaving-beta-form"
+        {{on "submit" (fn this.deactivateModal "formModalActive")}}
```

Comment:
to avoid confusion, could it make sense to create another function in the backing class, called `submitForm` and use it here? (even if technically what it does, for the purpose of the demo, is still deactivate the modal)

Reply:
sure, will do so